include/crm/common/output_internal.h

```diff
@@ -475,8 +475,10 @@ struct pcmk__output_s {
      *        plain text, console based formatters.
      *
      * \param[in] out The output functions structure.
+     * \param[in] end If true, output a newline afterwards.  This should
+     *                only be used the last time this function is called.
      */
-    void (*progress) (pcmk__output_t *out);
+    void (*progress) (pcmk__output_t *out, bool end);
```

Comment:
Ideally "end" would be part of the commit creating progress()

Reply:
Hah, yes, that was what I intended to do.